lib/capybara/spec/session/has_field_spec.rb

```diff
@@ -16,25 +16,29 @@
   context 'with value' do
     it "should be true if a field with the given value is on the page" do
       expect(@session).to have_field('First Name', :with => 'John')
+      expect(@session).to have_field('First Name', :with => /John/)
```

Comment:
All the added tests should probably check for substrings or something just to make sure the regexps are actually working